app/dashboard/models.py

```diff
@@ -944,6 +960,67 @@ def graph_timeseries_stacked(simulations, y_variables, energy_vector):
     return simulations_results
 
 
+def graph_timeseries_stacked_cpn(simulations, y_variables, energy_vector):
```

Comment:
Why not using `graph_timeseries_stacked()` and provide the y_variables you would like to plot?

Reply:
I wanted to leave the option open to have this graph be built differently (for example right now the excess dc and battery charge are filled instead of lines like in the original timeseries_stacked) and be able to adapt it according to feedback without having to mess around with the original open plan stacked graph